js/.husky/pre-commit

```diff
@@ -1,14 +1,4 @@
 #!/bin/sh
```

Comment:
The [migration guide](https://github.com/typicode/husky/releases/tag/v9.0.1) seems to imply we can remove the shebang as well.
```suggestion
```